### DISCUSSION TITLE
导入浏览器数据失败

### DISCUSSION THREAD

**Discussion by eamooooon** [shion-app/shion#16] [General]
如图
![image](https://github.com/shion-app/shion/assets/51506762/f0f84034-0112-48ce-87ec-10625f5f5545)

**Comment by hanaTsuk1** [comment]
在`帮助`->`打开开发者工具`，看一下报错

**Reply by eamooooon** [reply]
> 在`帮助`->`打开开发者工具`，看一下报错

![image](https://github.com/shion-app/shion/assets/51506762/fe864bb5-4abf-453b-88fc-055083b47037)

**Reply by hanaTsuk1** [reply]
这跟火狐浏览器有关

用powershell运行一下下面的命令
`explorer "$env:AppData\Mozilla\Firefox\Profiles"`
或是用文件管理器打开目录`C:\Users\{替换你的用户名}\AppData\Roaming\Mozilla\Firefox\Profiles`

在这个文件下面，应该有一个或多个文件夹，每个都应该代表你使用火狐浏览器时对应的用户账号，在其中找到**places.sqlite**这个文件

不知道为什么你这边报错了，正常情况是有这张表记录数据的，如果你不用火狐浏览器了可以把它们都删了
如果你正在使用火狐浏览器，且没有对其做卸载等影响文件操作的话，我需要花时间搜索类似的情况，进一步查明原因

**Reply by eamooooon** [reply]
> 这跟火狐浏览器有关
> 
> 用powershell运行一下下面的命令 `explorer "$env:AppData\Mozilla\Firefox\Profiles"` 或是用文件管理器打开目录`C:\Users\{替换你的用户名}\AppData\Roaming\Mozilla\Firefox\Profiles`
> 
> 在这个文件下面，应该有一个或多个文件夹，每个都应该代表你使用火狐浏览器时对应的用户账号，在其中找到**places.sqlite**这个文件
> 
> 不知道为什么你这边报错了，正常情况是有这张表记录数据的，如果你不用火狐浏览器了可以把它们都删了 如果你正在使用火狐浏览器，且没有对其做卸载等影响文件操作的话，我需要花时间搜索类似的情况，进一步查明原因

好了 把文件夹删了就OK了 我很久没用过火狐了可能是当时卸载的时候没有卸载干净
